deployments/bridges/westend-millau/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+# Exposed ports: 10616
+
+version: '3.5'
+services:
+  relay-headers-westend-to-millau:
+    networks:
+      - deployments_default
+      - deployments_external
+    image: paritytech/substrate-relay
+    entrypoint: /entrypoints/relay-headers-westend-to-millau-entrypoint.sh
+    volumes:
+      - ./bridges/westend-millau/entrypoints:/entrypoints
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
+    ports:
+      - "10616:9616"
+    depends_on:
+      - millau-node-alice
+
+  # Note: These are being overridden from the top level `monitoring` compose file.
+  grafana-dashboard:
+    environment:
+      VIRTUAL_HOST: grafana.millau.brucke.link,grafana.rialto.brucke.link
+      VIRTUAL_PORT: 3000
+      LETSENCRYPT_HOST: grafana.millau.brucke.link,grafana.rialto.brucke.link
+      LETSENCRYPT_EMAIL: admin@parity.io
+    volumes:
+       - ./bridges/westend-millau/dashboard/grafana/relay-westend-to-millau-headers-dashboard.json:/etc/grafana/provisioning/dashboards/relay-westend-to-millau-headers-dashboard.json
+
+  prometheus-metrics:
+    volumes:
+      - ./bridges/westend-millau/dashboard/prometheus/prometheus.yml:/etc/prometheus/prometheus-westend-millau.yml
+    depends_on:
+      - relay-headers-westend-to-millau
+
+networks:
+  deployments_external:
+    driver: bridge
+  deployments_default:
+    external: true
```

Comment:
Why do we need to specify networking options for this Compose deployment but not the others?

Reply:
Because it needs access to external node (so needs access to both defaul network and bridged network)

Reply:
I'm not convinced this is actually needed at all. What it does it makes all containers part of a single docker host-bridged network and we want isolation.
Are you sure it's not an issue with your local setup: https://docs.docker.com/network/bridge/#enable-forwarding-from-docker-containers-to-the-outside-world ?

Reply:
> it makes all containers part of a single docker host-bridged network and we want isolation

Why do you think so? I see that this particular container will have access two networks, of which one is bridged. We can't have isolated network and reach external node, can we?

Reply:
Actually, it seems to work for me, because deployments_default is also using bridge driver. But then what do you mean by "isolation"?

Reply:
Right, I meant isolation from the host machine, but perhaps I'm confusing `bridge` and `host` drivers. 
However you've changed `nginx-proxy` network as well, and I've tried this one to stay isolated from containers that don't need it, hence all containers ended up in both networks, I guess we can revert this back.

Reply:
Yes, I've changed nginx-proxy as well - it's already reverted after removal of that extra network.